src/hooks/mod.rs

```diff
@@ -76,6 +66,31 @@ fn evaluate_scripts(dir: &Path, scripts: &[String], engine: rhai::Engine) -> Res
     Ok(())
 }
 
+pub fn evaluate_script<T: Clone + 'static>(
+    liquid_object: liquid::Object,
+    script: &str,
+) -> Result<T, Box<rhai::EvalAltResult>> {
+    let mut conditional_evaluation_engine = rhai::Engine::new();
+
+    #[allow(deprecated)]
```

Comment:
Is there not yet way out of this deprecation? 
I'm a bit hesitant to let new functionality use deprecated features.

Can you provide a little context on why it make sense to use this very deprecated feature?

Reply:
not that I know of.

It is not a new one either - it's been there since we introduced `rhai`. They deprecatec the api because it *might* change according to their documentation.